scales-other.Rmd

```diff
@@ -169,6 +169,38 @@ base +
 
 For more information about manual scales see Section \@ref(scale-manual).
 
+## Line width {#scale-linewidth}
+
+The linewidth aesthetic, introduced in ggplot2 3.4.0, is used to control the width of lines. In earlier versions of ggplot2 the size aesthetic was used for this purpose, which caused some difficulty for complex geoms such as `geom_pointrange()` that contain both points and lines. For these geoms it's often important to be able to separately control the size of the points and the width of the lines. This is illustrated in the plots below. In the leftmost plot both the size and linewidth aesthetics are set at their default values. The middle plot increases the size of the points while leaving the linewidth unchanged, whereas the plot on the right increases the linewidth while leaving the point size unchanged. 
```

Comment:
I think it's worth also pointing out that you generally want line width to scale linearly with the value, but you want size to scale with the square-root of value so since we mostly perceive area of point, not radius.

Reply:
I guess we mostly say that above; so I think here we could just emphasise that we want a linear scale for linewidth.

Reply:
I've added a third example plot to the section that highlights this, and also illustrates the use of binned linewidth scales. It's designed to mirror the analogous discussion in the size scales section. I have mixed feelings about it though: in the "pro" column, it does cover the relevant issues and connects the linewidth and size sections nicely. But in the "con" column, it's an unusual way to use the linewidth aesthetic and the code snippet is a little long. Maybe that doesn't matter so much, though? The two previous examples are closer to typical use cases and both have short code snippets, so perhaps I'm being too picky